source/clear-linux/tutorials/machine-learning/machine-learning.rst

```diff
@@ -33,7 +40,7 @@ directory:
    sudo swupd bundle-add machine-learning-basic
```

Comment:
Could you please code both commands here as `.. code-blocks:: bash`?

Reply:
@krisdale Will you please change all other **input commands** to use same syntax as above in this document? 
`.. code-block:: bash `